test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -1970,46 +1971,46 @@ public void PackCommand_PackNewProject_CleanDeletesOnlyGeneratedNupkgAndNuspec()
         }
 
         [PlatformTheory(Platform.Windows)]
-        [InlineData("abc.txt",                  null,                               "content/abc.txt;contentFiles/any/netstandard1.4/abc.txt")]
```

Comment:
I think the whitespace here is on purpose to improve the readability of the test cases. 
Revert this please.

Reply:
Got them by formatting the doc. Reverted.